components/programs/ClassInstanceData.tsx

```diff
@@ -786,61 +800,34 @@ function workshop({
 }
 
 export const WorkshopInstances: ClassOrWorkshopInstance[] = [
-  //  workshop({
-  //     course: Courses.python_workshop,
-  //     dateString: "2021-01-25T17:30:00-08:00",
-  //     minutes: 90,
-  //     signupUrl: "https://www.tfaforms.com/4874286",
-  //     who: "7th - 12th graders",
-  //   }),
-  //   workshop({
-  //     course: Courses.deep_dream,
-  //     dateString: "2021-02-01T15:00:00-08:00",
-  //     minutes: 90,
-  //     signupUrl: "https://www.tfaforms.com/4879388",
-  //     who: "7th - 12th graders",
-  //     buttonExtra: <>New workshop!</>,
-  //   }),
   workshop({
-    course: Courses.p5js_workshop,
-    dateString: "2021-03-04T16:00:00-08:00",
-    minutes: 90,
-    signupUrl: "https://www.tfaforms.com/4887060",
-    who: "7th - 12th graders",
+    course: Courses.cover_letter_workshop,
+    dateString: "2021-03-31T15:00:00-07:00",
+    minutes: 120,
+    signupUrl: " https://www.tfaforms.com/4887087",
+    who: "14 - 24 year olds",
   }),
   workshop({
-    course: Courses.video_sensing_workshop,
-    dateString: "2021-03-16T15:00:00-07:00",
+    course: Courses.p5js_workshop,
+    dateString: "2021-04-15T16:00:00-08:00",
     minutes: 90,
-    signupUrl: "https://www.tfaforms.com/4887340",
+    signupUrl: "https://www.tfaforms.com/4892574",
     who: "7th - 12th graders",
   }),
   workshop({
-    course: Courses.design_business_workshop,
-    dateString: "2021-03-22T15:00:00-07:00",
+    course: Courses.deep_dream,
+    dateString: "2021-04-22T16:00:00-08:00",
     minutes: 90,
-    signupUrl: "https://www.tfaforms.com/4887067",
+    signupUrl: "https://www.tfaforms.com/4892553",
     who: "7th - 12th graders",
   }),
   workshop({
-    course: Courses.cover_letter_workshop,
-    dateString: "2021-03-31T15:00:00-07:00",
+    course: Courses.job_hunt,
+    dateString: "2021-04-28T16:00:00-08:00",
     minutes: 120,
-    signupUrl: " https://www.tfaforms.com/4887087",
-    who: "14 - 24 year olds",
+    signupUrl: "https://www.tfaforms.com/4892571",
+    who: "7th - 12th graders",
```

Comment:
this should be 14-24-year-olds since it's a career prep workshop!

Reply:
Fixed and updated times for the workshop 👍